vault/resource_pki_secret_backend_config_issuers_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccPKISecretBackendConfigIssuers_basic(t *testing.T) {
```

Comment:
Is it possible to mark this test with `t.Parallel()`?

Reply:
Sure thing!

Reply:
I don't see any harm in doing so. The only caveat is that tests will fail if they do not properly handle conditions to allow multiple tests to occur against the same resource or service (e.g. random naming).

FWIW, the TF SDK also provides [ParallelTest](https://github.com/hashicorp/terraform-plugin-sdk/blob/aa0b6acece22b385dc9d5f4d00313ab9e88c7e10/helper/resource/testing.go#L679). There are some pros and cons listed here: https://github.com/hashicorp/terraform-plugin-sdk/issues/1

Reply:
Cool! I marked it as `t.Parallel()` at the start of the test and double checked that the resource and path names are dynamically generated. As long as CI passes I think we should be good

Reply:
CI is all green, looks like we're good! Thanks folks